### PR TITLE
added goo

### PR DESCRIPTION
**Pull request by sulphydryl**

### CODE REVIEW

**Comment by sulphydryl** [comment]
pls merge ty

**Comment by sulphydryl** [comment]
oops

File: fun/README.md [hunk 1]
```diff
@@ -246,5 +246,6 @@ Keeping track of amusing translations I encountered
 ### Uncategorized 2
 * hehhh-- (へーー) "The woolly mammoth was around when the pyramids were being built." -> "hehhh-- no way"
 * 気持ち悪い "(sees a trypophobia-inducing thing)" -> "omggg ewww that's so 気持ち悪い"
-* 勿体ない "All the strawberries rotted. I'm gonna have to throw them all out" -> "That's so 勿体ない..."
+* 勿体ない "All the strawberries rotted. I'm gonna have to throw them all out" -> "That's so 勿体ない..."]
+* 具（ぐ）”Wow this pasta has a lot ぐ in it"
```
Comment (atskae) [inline fun/README.md]: Please add a comma after `"Wow"`

**Comment by sulphydryl** [comment]
done

**Review by atskae — APPROVED** [review]
Looks good to me, ありがっつ!!

**Pull request merged**